app/dashboard/migrations/0092_auto_20200318_0001.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 2.2.4 on 2020-03-18 00:01
+
+import django.contrib.postgres.fields
```

Comment:
can we rebase and recreate this migration? now 0092 is already used by other migration

Reply:
sure